notify/webhook/webhook.go

```diff
@@ -96,6 +112,8 @@ func (n *Notifier) Notify(ctx context.Context, alerts ...*types.Alert) (bool, er
 		Data:            data,
 		GroupKey:        groupKey.String(),
 		TruncatedAlerts: numTruncated,
+		FiringAlerts:    numFiring,
```

Comment:
If we're concerned about efficiency, I'd rather add some helper functions to the `template` package.
```suggestion
		FiringAlerts:    len(alerts.Firing()),
```